src/test/java/io/aiven/kafka/connect/transforms/HashTest.java

```diff
@@ -211,7 +212,24 @@ void fieldName_EmptyStringValue(final String hashFunction) {
         final Hash<SinkRecord> transform = transformation(FIELD, true, hashFunction);
         final SinkRecord result = transform.apply(originalRecord);
         final String newValue = hash(hashFunction, EMPTY_FIELD_VALUE);
-        assertEquals(setNewValue(originalRecord, newValue), result);
+        assertNotEquals(setNewValue(originalRecord, newValue), result);
+    }
+
+    @Test
+    void fieldNameHashOnlyTheValue() {
```

Comment:
Instead of a separate test, let's have some `UNAFFECTED_FIELD` added into all tests with `Struct`s.